node/resolvers/Mutations/Organizations.ts

```diff
@@ -18,6 +18,7 @@ import B2BSettings from '../Queries/Settings'
 import CostCenters from './CostCenters'
 import MarketingTags from './MarketingTags'
 import checkConfig from '../config'
+import { sendUpdateOrganizationMetric } from '../../utils/metrics/organization'
```

Comment:
I guess the changes on this file should not be on this PR? This is part of the of this other [PR](https://github.com/vtex-apps/b2b-organizations-graphql/pull/127)?